app/views/devise/two_factor_authentication/show.html.slim

```diff
@@ -11,7 +11,7 @@ p
 
 = form_tag([:user, :two_factor_authentication], method: :put, role: 'form') do
   .mb2
-    = label_tag 'code', raw('<abbr title="required">*</abbr> ') + t('forms.two_factor.code')
```

Comment:
Just came across this:

> It must be clearly stated when a field is required to be filled in. The term required can be used in the field label. Color and font differences must not be the only indication that a field is required. Alternatively, a symbol can be used to indicate that a particular field is required. By convention, the required symbol is an asterisk (*). When such a symbol is used, its purpose must be explained in the form’s instructions. 

http://www.section508.va.gov/support/html/10_1a.asp